notebooks/search/01-keyword-querying-filtering.ipynb

```diff
@@ -1,14 +1,5 @@
 {
   "cells": [
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "0wgbLWl2udLQ"
-      },
-      "source": [
-        "[Quick Start](https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/search/00-quick-start.ipynb) || **Keyword Querying Filtering** || [Hybrid search with RRF](https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/search/02-hybrid-search.ipynb) || [ELSER](https://github.com/elastic/elasticsearch-labs/blob/main/notebooks/search/03-ELSER.ipynb)"
-      ]
-    },
```

Comment:
@joshdevins is just a drive-by cleanup? :)

Reply:
Yeah, I wanted to normalise all the notebooks and this one stood out as having a header that none of the others had.